src/db/seeders/20210420144709-add-users.ts

```diff
@@ -0,0 +1,31 @@
+import * as Sequelize from 'sequelize';
+
+import { UserAttributes, UserInstance } from '../models/user';
+
+import dbFactory, * as models from '../models';
+
+const ADMINS = [
+  { name: 'Ben', email: 'anonyben@gmail.com' },
+  { name: 'Matt', email: 'matthew.hampel@gmail.com' },
```

Comment:
could I be matt@matth.org?
Oh wait if it needs to be gmail, then mahatm@gmail.com